ansible/golden_config_db/smartswitch_t1-28-lag.json

```diff
@@ -0,0 +1,111 @@
+{
+    "CHASSIS_MODULE": {
+        "DPU0": {
+            "admin_status": "down"
+        },
+        "DPU1": {
+            "admin_status": "down"
+        },
+        "DPU2": {
+            "admin_status": "down"
+        },
+        "DPU3": {
+            "admin_status": "down"
+        },
+        "DPU4": {
+            "admin_status": "down"
+        },
+        "DPU5": {
+            "admin_status": "down"
+        },
+        "DPU6": {
+            "admin_status": "down"
+        },
+        "DPU7": {
+            "admin_status": "down"
+        }
+    },
+    "DHCP_SERVER_IPV4": {
```

Comment:
For Nvidia platform we have 4 only DPUs, need to generate golden config based on the platform type.

Reply:
Hi @congh-nvidia , could you provide the Nvidia platfrom string?

Reply:
Hi @prabhataravind , does other platfrom are not using 8 dpus? Is it just nvidia using 4?

Reply:
> Hi @congh-nvidia , could you provide the Nvidia platfrom string?

This is the additional config we are currently using with the config generated from t1-28-lag minigraph. The "PORT" section for the npu-dpu interfaces is also not generated from minigraph, so we need to apply it with this additional config. And we don't have the "CHASSIS_MODULE" section because the PMON for smartswitch has not been integrated into our internal images yet.
Our platform is: **x86_64-nvidia_sn4280-r0**
HWSKU is: **Mellanox-SN4280-O28**
```
{
    "DEVICE_METADATA": {
        "localhost": {
            "subtype": "SmartSwitch"
        }
    },
    "FEATURE": {
        "dhcp_server": {
            "state": "enabled"
        },
        "dhcp_relay": {
            "state": "enabled"
        }
    },
    "MID_PLANE_BRIDGE": {
        "GLOBAL": {
            "bridge": "bridge-midplane",
            "ip_prefix": "169.254.200.254/24"
        }
    },
    "DHCP_SERVER_IPV4": {
        "bridge-midplane": {
            "customized_options": [
                "option60",
                "option223"
            ],
            "gateway": "169.254.200.254",
            "lease_time": "900",
            "mode": "PORT",
            "netmask": "255.255.255.0",
            "state": "enabled"
        }
    },
    "DHCP_SERVER_IPV4_PORT": {
        "bridge-midplane|dpu0": {
            "ips": [
                "169.254.200.1"
            ]
        },
        "bridge-midplane|dpu1": {
            "ips": [
                "169.254.200.2"
            ]
        },
        "bridge-midplane|dpu2": {
            "ips": [
                "169.254.200.3"
            ]
        },
        "bridge-midplane|dpu3": {
            "ips": [
                "169.254.200.4"
            ]
        }
    },
    "DPUS": {
        "dpu0": {
            "midplane_interface": "dpu0",
            "interface": {
                "Ethernet224": "Ethernet0"
            },
            "bus_info": "08:00.0"
        },
        "dpu1": {
            "midplane_interface": "dpu1",
            "interface": {
                "Ethernet232": "Ethernet0"
            },
            "bus_info": "07:00.0"
        },
        "dpu2": {
            "midplane_interface": "dpu2",
            "interface": {
                "Ethernet240": "Ethernet0"
            },
            "bus_info": "01:00.0"
        },
        "dpu3": {
            "midplane_interface": "dpu3",
            "interface": {
                "Ethernet248": "Ethernet0"
            },
            "bus_info": "02:00.0"
        }
    },
    "PORT": {
        "Ethernet224": {
            "admin_status": "up",
            "speed": "400000"
        },
        "Ethernet232": {
            "admin_status": "up",
            "speed": "400000"
        },
        "Ethernet240": {
            "admin_status": "up",
            "speed": "400000"
        },
        "Ethernet248": {
            "admin_status": "up",
            "speed": "400000"
        }
    }
}
```

Reply:
Hi @congh-nvidia , we will use a new topo for such support. And we suggest to add number of DPU into topo xml file. So we don't need to check platform string.